### ISSUE TITLE
dxf Export does not work Since Version 3.12

### ISSUE THREAD

**Issue by gafzgarrr** [qgis/QGIS#39799]
dxf Export from QGIS 3.10 worked without Problems.
On higher Versions (Tested on 3.12 an 3.14) the dxf Export does not work.
The dxf File is Empty or only the Descriptions were exported.

**Comment by gioman** [comment]
@gafzgarrr can you be more specific? I just tried to export a vector to DXF (using project > import/export > export project to DXF) and seems to have worked.

**Comment by gioman** [comment]
Related/duplicate of https://github.com/qgis/QGIS/issues/37202 ?

**Comment by gafzgarrr** [comment]
@gioman thanks for responding. 
I do it the same way (using project ....) 

I get an dxf-file but there are no Lines or Points inside. The dxf File is empt or (from time to time) there are short Lines.
I use AutoCAD Map2017 for opening the dxf files.
I tried with 2 other PC´s always the same.

QGIS 3.10 has no issues, QGIS 3.12 an 3.14 has the Problems.

It always looks like the Picture
![2020-11-05 08_09_57-Autodesk AutoCAD Map 3D 2017 -  Borna_gesamt_1 dxf](https://user-images.githubusercontent.com/59369853/98208948-723d8580-1f3e-11eb-917a-65cc85966bcf.jpg)

**Comment by gioman** [comment]
> I get an dxf-file but there are no Lines or Points inside.

@gafzgarrr attach a sample of your data.

**Comment by spad1991** [comment]
I provide two samples of the same file exported from 3.10.3 LTR and 3.16.0.
The labels are exported correctly but all the symbols are collapsed to a single point.
[SAMPLES.zip](https://github.com/qgis/QGIS/files/5561027/SAMPLES.zip)

**Comment by gioman** [comment]
>  I provide two samples of the same file exported from 3.10.3 LTR and 3.16.0.

@spad1991 you must add the **input** data also, thanks.

**Comment by spad1991** [comment]
> 
> 
> > I provide two samples of the same file exported from 3.10.3 LTR and 3.16.0.
> 
> @spad1991 you must add the **input** data also, thanks.

Sorry but I'm not allowed to share the input data. I can tell you that the points are from a .csv file with EPSG:4326 - WGS 84 coordinates. I belive you can reproduce the error with any coordinates structure (I have tried WKT and X,Y columns).

**Comment by gioman** [comment]
> Sorry but I'm not allowed to share the input data. I can tell you that the points are from a .csv file with EPSG:4326 - WGS 84 coordinates. I belive you can reproduce the error with any coordinates structure (I have tried WKT and X,Y columns).

@gafzgarrr can you provide/attach such sample?

**Comment by spad1991** [comment]
@gioman I found an approved project I can share to show the problem. I cannot upload the background, it's too heavy and also irrelevant to the issue (last time it was a poly shapefile, wich exported correctly, this time it is a .tif raster that is not exportable in dxf, no change in the point geometry behaviour).
[SAMPLES.zip](https://github.com/qgis/QGIS/files/5588785/SAMPLES.zip)

**Comment by gioman** [comment]
> @gioman I found an approved project I can share to show the problem. I cannot upload the background, it's too heavy and also irrelevant to the issue (last time it was a poly shapefile, wich exported correctly, this time it is a .tif raster that is not exportable in dxf, no change in the point geometry behaviour).
> [SAMPLES.zip](https://github.com/qgis/QGIS/files/5588785/SAMPLES.zip)

@spad1991 just tested your sample on 3.10 and 3.16 and I get identical results.

**Comment by spad1991** [comment]
> 
> 
> > @gioman I found an approved project I can share to show the problem. I cannot upload the background, it's too heavy and also irrelevant to the issue (last time it was a poly shapefile, wich exported correctly, this time it is a .tif raster that is not exportable in dxf, no change in the point geometry behaviour).
> > [SAMPLES.zip](https://github.com/qgis/QGIS/files/5588785/SAMPLES.zip)
> 
> @spad1991 just tested your sample on 3.10 and 3.16 and I get identical results.

So is it because of something in my export settings? 
![Cattura](https://user-images.githubusercontent.com/74669420/100102230-987e8300-2e63-11eb-80f6-46d348bf90f3.PNG)

**Comment by gioman** [comment]
> So is it because of something in my export settings?

you are exporting using the 32632 CRS, but your project/data is in 4326 and I don't think that this parameter allows you do any reprojection, it is supposed you to choose the correct CRS, in this case 4326.

**Comment by gioman** [comment]
Closing for lack of feedback.

**Comment by tmcgill26** [comment]
I am not sure if I should post here or somewhere else since this issue has been closed, but I have experienced the same problem (the current version is 3.16.2). 

I do not know if the change in behavior that occurred after 3.10 (I skipped 3.12 and went straight to 3.14) was intentional or accidental, but I much prefer the older behavior. If the change was intentional, I would like to convert this issue to a feature request.

My circumstances can be described as:
1. Project CRS is defined.
2. Web layers contained in the project have a properly defined CRS that is different from the project CRS.
3. Initiate the [Project] [Import/Export] [Export Project to DXF] dialog to export a DXF. 
4. Set the [CRS] setting to the project CRS (mine is set to this by default or precedent).
5. Export the DXF.
6. Open the DXF in AutoCAD Civil 3D

When I do this in version 3.10, all layers are exported using the project CRS (the web layers have been reprojected and everything looks just like it did on the screen in QGIS). When I do this in version 3.16, all layers are located in their original and un-reprojected locations and they do not line up on the screen.
